McsaMeetsMailer/BusinessLogic/FullScheduleEmailBuilder.cs

```diff
@@ -0,0 +1,42 @@
+﻿using McsaMeetsMailer.Models;
+using McsaMeetsMailer.Utils.Html;
+using System.Collections.Generic;
+using System.Linq;
+
+namespace McsaMeetsMailer.BusinessLogic
+{
+  public static class FullScheduleEmailBuilder
+  {
+    public static string Build(IEnumerable<MeetDetailsModel> meetDetails, IHtmlBuilder htmlBuilder)

```

Comment:
Will ultimately need to order columns as per the original sheet.

Reply:
Added ticket to rethink use of dictionary.